cli/exchange/user.go

```diff
@@ -89,6 +98,14 @@ type UserExchangePatchAdmin struct {
 }
 
 func UserSetAdmin(org, userPwCreds, user string, isAdmin bool) {
+	msgPrinter := i18n.GetMessagePrinter()
+
+	if emailIsOptional, _ := checkExchangeVersionForOptionalUserEmail(org, userPwCreds); emailIsOptional {
```

Comment:
Again, i don't think it is important to check `emailIsOptional` here

Reply:
Same as above.`emailIsOptional == true` means user has exchange version later than 2.61.0, then I add my new checks. If `emailIsOptional == false`, it will use the old code path

Reply:
I agree with Bruce, dont need the exchange version mixed into the check.